src/tool/subcommands/api_cmd/report.rs

```diff
@@ -0,0 +1,437 @@
+// Copyright 2019-2025 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use super::ReportMode;
+use crate::rpc;
+use crate::rpc::FilterList;
+use ahash::{HashMap, HashMapExt};
+use serde::{Deserialize, Serialize};
+use similar::{ChangeTag, TextDiff};
+use std::path::Path;
+use std::time::Instant;
+use tabled::{builder::Builder, settings::Style};
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+struct PerformanceMetrics {
+    total_duration_ms: u128,
+    average_duration_ms: u128,
+    min_duration_ms: u128,
+    max_duration_ms: u128,
+    test_count: usize,
+}
+
+// Add a helper function to calculate performance metrics
+impl PerformanceMetrics {
+    pub fn from_durations(durations: &[u128]) -> Option<Self> {
+        if durations.is_empty() {
+            return None;
+        }
+
+        let total_duration_ms = durations.iter().sum();
+        let test_count = durations.len();
+        let average_duration_ms = total_duration_ms / test_count as u128;
+        let min_duration_ms = *durations.iter().min().unwrap();
+        let max_duration_ms = *durations.iter().max().unwrap();
+
+        Some(Self {
+            total_duration_ms,
+            average_duration_ms,
+            min_duration_ms,
+            max_duration_ms,
+            test_count,
+        })
+    }
+}
+
+/// Details about a successful test instance
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "snake_case")]
+struct SuccessfulTest {
+    /// The parameters used for this test
+    request_params: serde_json::Value,
+
+    /// Forest node response
+    forest_status: String,
+
+    /// Lotus node response  
+    lotus_status: String,
```

Comment:
If it's a response, it'd name the fields `[forest/lotus]_response`. I'd expect a field named `status` to be an integer (or enum encompassing it) like 200 or 418.